src/main/java/io/qameta/allure/bamboo/AllureDownloader.java

```diff
@@ -68,6 +73,20 @@
         return Optional.empty();
     }
 
+    private void addExecutablePermissions(Path allureHome) {
+        Set<PosixFilePermission> perms = new HashSet<>();
+        perms.add(PosixFilePermission.OWNER_EXECUTE);
+        perms.add(PosixFilePermission.OWNER_READ);
+        perms.add(PosixFilePermission.OWNER_WRITE);
+        Path binaries = allureHome.resolve("bin");
+        try {
+            Files.setPosixFilePermissions(binaries.resolve("allure"), perms);
```

Comment:
this code will throw `UnsupportedOperationException` in case of OS that not supports Posix file permissions. So there are no need to set posix file permissions for bat file as well